packages/core/src/test/awsService/cloudWatchLogs/registry/liveTailSession.test.ts

```diff
@@ -65,6 +83,36 @@ describe('LiveTailSession', async function () {
         )
     })
 
+    it('startLiveTail sends command to CWL client with AbortSignal', async function () {
```

Comment:
Looks like a lot of code to test very little. The test description sounds like it's testing that a parameter is passed somewhere.